docs/utilities/parser.md

```diff
@@ -169,6 +155,14 @@ def my_function():
 		}
 ```
 
+#### Arbitrary data model parsing
```

Comment:
use `primitives` instead of `arbitrary`, as `Enum` or `SQLAlchemy Models` wouldn't work here